Jenkinsfile

```diff
@@ -11,60 +11,71 @@ pipeline {
             parallel {
                 stage("StaticAnalysis") {
                     steps {
-                        node("slave-sbt") {
-                            checkout scm
-                            sh 'sbt clean scalafmtCheck scalafmtSbtCheck test:scalafmtCheck scapegoat'
-                        }
+                        checkout scm
+                        sh 'sbt clean scalafmtCheck scalafmtSbtCheck test:scalafmtCheck scapegoat'
                     }
                 }
                 stage("Tests/Coverage") {
                     steps {
-                        node("slave-sbt") {
-                            checkout scm
-                            sh "sbt clean coverage test coverageReport coverageAggregate"
-                            sh "curl -s https://codecov.io/bash >> ./coverage.sh"
-                            sh "bash ./coverage.sh -t `oc get secrets codecov-secret --template='{{.data.nexus_admin}}' | base64 -d`"
-                        }
+                        checkout scm
+                        sh "sbt clean coverage test coverageReport coverageAggregate"
+                        sh "curl -s https://codecov.io/bash >> ./coverage.sh"
+                        sh "bash ./coverage.sh -t `oc get secrets codecov-secret --template='{{.data.nexus_admin}}' | base64 -d`"
                     }
                 }
             }
+            stage("Build Image & Deploy") {
+                steps {
+                    checkout scm
+                    sh 'sbt universal:packageZipTarball'
+                    sh "mv modules/service/target/universal/admin-service-*.tgz ./admin-service.tgz"
+                    sh "oc start-build admin-build --from-file=admin-service.tgz --follow"
+                    openshiftTag srcStream: 'admin', srcTag: 'latest', destStream: 'admin', destTag: version.substring(1), verbose: 'false'
+                    sh "oc project bbp-nexus-dev"
+                    sh "oc scale statefulset admin --replicas=0"
+                    sleep 2
+                    sh "oc scale statefulset admin --replicas=1"
+                    sh '''
+                        while true; do
+                            replicas=$(oc get statefulset/admin -o=jsonpath='{.status.readyReplicas}')
+                            ((replicas > 1)) && break
+                            sleep 1
+                        done
+                    '''
+                    build job: 'nexus-tests', parameters: [ booleanParam(name: 'run', value: true) ]
```

Comment:
it has to run the tests. Something like `sh "sbt clean test`
I don't know what this `run` is doing, but maybe is the same